kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -14,6 +14,10 @@ ssui_root="$repos_root/manageiq-ui-self_service"
 # Symlink manageiq to the vmdb root so that the SSUI can build into it
 ln -vs $vmdb_root $manageiq_root
 
+# Replace the log directory with the larger logvol
+rm -rf $vmdb_root/log
+ln -vs $app_root/vmdb_log $vmdb_root/log
```

Comment:
If we mount the log directory directly at vmdb/log the  git clone of vmdb fails with vmdb already exists.